stdlib/public/runtime/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ add_swift_library(swiftRuntime OBJECT_LIBRARY TARGET_LIBRARY
   C_COMPILE_FLAGS ${swift_runtime_library_compile_flags}
   LINK_FLAGS ${swift_runtime_linker_flags}
   INSTALL_IN_COMPONENT never_install
-  TARGET_SDKS ANDROID CYGWIN FREEBSD LINUX)
+  TARGET_SDKS ANDROID CYGWIN FREEBSD LINUX WINDOWS)
```

Comment:
As part of a separate PR, I think a cleaner way to do this would be to define lists named `OBJC_INTEROP_PLATFORMS` and `NO_OBJC_INTEROP_PLATFORMS`, and use them instead of `ALL_APPLE_PLATFORMS`.

---

Here's a zany idea: to support projects like [mulle-objc](https://mulle-objc.github.io), we allow the lists to be configurable via the build script and CMake. It would be really cool if I could compile for macOS without Objective-C support enabled, or compile for Linux with Objective-C interop:

```
$ utils/build-script --target-no-objc-sdks macos
$ utils/build-script --target-objc-sdks linux,freebsd
```

If this sounds like something the core team would be interested in, I'd love to submit a PR for it! /cc @gottesmm